src/types/snakvalue.ts

```diff
@@ -0,0 +1,76 @@
+import type { EntityId, EntityType } from './entity.js'
+import type { WmLanguageCode } from './options.js'
+
+export type SnakValue =
+  | GlobecoordinateSnakValue
+  | MonolingualTextSnakValue
+  | QuantitySnakValue
+  | StringSnakValue
+  | TimeSnakValue
+  | WikibaseEntityIdSnakValue
+
+/** @deprecated use TimeSnakValue */
+export type ClaimSnakTimeValue = TimeSnakValue
+/** @deprecated use QuantitySnakValue */
+export type ClaimSnakQuantity = QuantitySnakValue
+/** @deprecated use StringSnakValue */
+export type ClaimSnakString = StringSnakValue
+/** @deprecated use WikibaseItemSnakValue */
+export type SnakEntityValue = WikibaseEntityIdSnakValue
+/** @deprecated use WikibaseItemSnakValue */
+export type ClaimSnakWikibaseItem = WikibaseEntityIdSnakValue
```

Comment:
oups… looks like the deprecated hint points to the wrong object name. Its easy to find out when looking into the code so not that important to fix?